### DISCUSSION TITLE
API keys added but none verified

### DISCUSSION THREAD

**Discussion by ghost** [matamorphosis/Scrummage#44] [Q&A] [answered]
I have added multiple API keys but none of them have been verified does it have something to do with running it on a virtual machine?

**Comment by matamorphosis** [comment]
Hi @R3CON6,

If you please elaborate more by confirming/responding to the following:

1. Which API Keys you have added (Just the names of the plugins is fine), the method you used to add the API keys, and please confirm all fields have been filled out.
2. You are verifying the plugins using the `https://<SCRUMMAGE-IP>:5000/tasks/inputs/check` URI.
3. By Virtual Machine, I presume you mean the pre-built docker image?

If you could also supply screenshots of the issue, that would be great.

Thanks,

Matt

**Reply by ghost** [reply]
Thanks for responding Matt
1. API keys I entered: I tried both ways of entering them first from web ui then from .config.json: All fields appear to be filled out 
bitcoinabuse
emailrep
greynoisesearch
hunter
hybridanalysis
intelligencex
ipstack
leaklookup
pulsedive
shodan
urlscan
virustotal
whatcms
![image](https://user-images.githubusercontent.com/71594033/148477132-bca19809-a426-412a-be15-892d7c9db744.png)

2. verifying plugins shows none valid
![image](https://user-images.githubusercontent.com/71594033/148477298-5b94bc00-14c1-4f66-a3ff-a9570d33ac17.png)

3. By virtual machine I mean running this on a kali VM on a windows host using VMware workstion

**Comment by matamorphosis** [comment, marked as answer]
Hey @R3CON6,

Thanks for the further info, there was a bug as a result of some recent changes which have been giving me some grief. A series of fixes have been made which should now mean tasks can be configured and run.

For tracking I set up an [issue request](https://github.com/matamorphosis/Scrummage/issues/45).

You will have to update your local copy of the repo in order for the fix to work.

Let me know if there are any further issues.

**Reply by ghost** [reply]
everything is working now thank you
